core/src/main/scala/org/http4s/Status.scala

```diff
@@ -76,102 +77,123 @@ object Status {
-  val SeeOther = register(Status(303, "See Other"))
-  val NotModified = register(Status(304, "Not Modified", isEntityAllowed = false))
-  val UseProxy = register(Status(305, "Use Proxy"))
-  val TemporaryRedirect = register(Status(307, "Temporary Redirect"))
-  val PermanentRedirect = register(Status(308, "Permanent Redirect"))
-
-  val BadRequest = register(Status(400, "Bad Request"))
-  val Unauthorized = register(Status(401, "Unauthorized"))
-  val PaymentRequired = register(Status(402, "Payment Required"))
-  val Forbidden = register(Status(403, "Forbidden"))
-  val NotFound = register(Status(404, "Not Found"))
-  val MethodNotAllowed = register(Status(405, "Method Not Allowed"))
-  val NotAcceptable = register(Status(406, "Not Acceptable"))
-  val ProxyAuthenticationRequired = register(Status(407, "Proxy Authentication Required"))
-  val RequestTimeout = register(Status(408, "Request Timeout"))
-  val Conflict = register(Status(409, "Conflict"))
-  val Gone = register(Status(410, "Gone"))
-  val LengthRequired = register(Status(411, "Length Required"))
-  val PreconditionFailed = register(Status(412, "Precondition Failed"))
-  val PayloadTooLarge = register(Status(413, "Payload Too Large"))
-  val UriTooLong = register(Status(414, "URI Too Long"))
-  val UnsupportedMediaType = register(Status(415, "Unsupported Media Type"))
-  val RangeNotSatisfiable = register(Status(416, "Range Not Satisfiable"))
-  val ExpectationFailed = register(Status(417, "Expectation Failed"))
-  val UnprocessableEntity = register(Status(422, "Unprocessable Entity"))
-  val Locked = register(Status(423, "Locked"))
-  val FailedDependency = register(Status(424, "Failed Dependency"))
-  val UpgradeRequired = register(Status(426, "Upgrade Required"))
-  val PreconditionRequired = register(Status(428, "Precondition Required"))
-  val TooManyRequests = register(Status(429, "Too Many Requests"))
-  val RequestHeaderFieldsTooLarge = register(Status(431, "Request Header Fields Too Large"))
-  val UnavailableForLegalReasons = register(Status(451, "Unavailable For Legal Reasons"))
-
-  val InternalServerError = register(Status(500, "Internal Server Error"))
-  val NotImplemented = register(Status(501, "Not Implemented"))
-  val BadGateway = register(Status(502, "Bad Gateway"))
-  val ServiceUnavailable = register(Status(503, "Service Unavailable"))
-  val GatewayTimeout = register(Status(504, "Gateway Timeout"))
-  val HttpVersionNotSupported = register(Status(505, "HTTP Version not supported"))
-  val VariantAlsoNegotiates = register(Status(506, "Variant Also Negotiates"))
-  val InsufficientStorage = register(Status(507, "Insufficient Storage"))
-  val LoopDetected = register(Status(508, "Loop Detected"))
-  val NotExtended = register(Status(510, "Not Extended"))
-  val NetworkAuthenticationRequired = register(Status(511, "Network Authentication Required"))
+  val Continue: Status = register(Status(100, "Continue", isEntityAllowed = false))
```

Comment:
This doesn't hurt the singleton type tricks we use in the DSL?  I'm trying to think why these might not have been ascribed before.  If it didn't break things, :+1:

Reply:
It appears to be alright, so I guess we're lucky.